examples/tutorials/analysis-3d/non_detected_source.py

```diff
@@ -0,0 +1,254 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+Computing flux upper limits
+============
+
+Explore how to deal with flux upper limits for a non-detected source.
+
+Prerequisites
+-------------
+
+It is advisable to understand the general Gammapy modelling and fitting
```

Comment:
```suggestion
It is advisable to understand the general Gammapy modeling and fitting
```
I think we use the american spelling most of the time.